person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  attr_accessor :age, :name
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def of_age?
+    @age >= 18
+  end
+  private :of_age?
+
+  def can_use_services?
+    is_of_age? || parent_permission
+  end
```

Comment:
- [ ] On line 18 you are calling the method `is_of_age?` to verify the person's age but the method's correct name is `of_age?` defined on line 12. Please fix this issue by correcting the method name.

Reply:
Oh, thank you so much for noticing me.